README.md

```diff
@@ -1,19 +1,55 @@
 Editing Task Force
 =================
 
-These are specifications and explainers related to the work of the Editing Task Force. 
+Editing Task Force is considered to be a part of [W3C WebApps Working Group](https://w3c.github.io/webappswg/). 
 
-**Actively developed specs:**
+Its purpose is to gather and incubate emerging issues in the area of rich text editor development. 
```

Comment:
Because I've been watching a lot Star Trek: 

```suggestion
Its mission is to gather and incubate emerging ideas as they pertain to rich text editor development. 
```